go/master/service.go

```diff
@@ -34,27 +34,28 @@ type Chunk struct {
 // Task is the basic unit of data instances assigned to trainers.
 type Task struct {
 	ID     int
+	Epoch  int
```

Comment:
Maybe change like this is more consistent with `func (s *Service) TaskFailed(taskID TaskID, dummy *int) error`:
```go
type TaskMeta struct {
  ID int
  Epoch int
}

type Task struct {
  TaskMeta Meta
  Chunks []Chunk
}

func (s *Service) TaskFailed(meta TaskMeta, dummy *int) error {
}
```